src/p2p/net_node.inl

```diff
@@ -1338,7 +1470,11 @@ namespace nodetool
       return true;
     if (get_incoming_connections_count() == 0)
     {
-      if (m_hide_my_port || m_config.m_net_config.max_in_connection_count == 0)
+      bool incoming_disabled = true;
+      for (const auto& zone : m_network_zones)
+        incoming_disabled &= (zone.second.m_config.m_net_config.max_in_connection_count == 0);
```

Comment:
Since you can have incoming connections from Tor where you can't sync, you probably want to keep this to the sync-able zones (if, you want the warning if all zones but tor have 0 max).

Reply:
Reverted this to only inspect the public zone.